### PR TITLE
Update Set-MpPreference.md

### PR DESCRIPTION
**Pull request by msbemba**
Added AllowSwitchToAsyncInspection parameter

fixes https://github.com/MicrosoftDocs/windows-powershell-docs/issues/3252

### CODE REVIEW

File: docset/winserver2016-ps/defender/Set-MpPreference.md
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/defender/Set-MpPreference.md:107]:
```suggestion
Specifies whether to enable a performance optimization that allows synchronously inspected network flows to switch to async inspection once they have been checked and validated.
```

File: docset/winserver2019-ps/defender/Set-MpPreference.md
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:90]:
```suggestion
Specifies whether to enable a performance optimization that allows synchronously inspected network flows to switch to async inspection once they have been checked and validated.
```

File: docset/winserver2022-ps/defender/Set-MpPreference.md
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/defender/Set-MpPreference.md:151]:
```suggestion
Specifies whether to enable a performance optimization that allows synchronously inspected network flows to switch to async inspection once they have been checked and validated.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tiburd** [comment]
waiting for review by @get-itips

**Review by get-itips — APPROVED** [review]

**Pull request merged**
